boards/arm/frdm_k64f/doc/index.rst

```diff
@@ -241,16 +241,29 @@ configured by default to use the :ref:`opensda-daplink-onboard-debug-probe`.
 Early versions of this board have an outdated version of the OpenSDA bootloader
 and require an update. Please see the `DAPLink Bootloader Update`_ page for
 instructions to update from the CMSIS-DAP bootloader to the DAPLink bootloader.
+The version 0254 is known to work well to flash and debug.
 
 .. tabs::
 
    .. group-tab:: OpenSDA DAPLink Onboard (Recommended)
 
-        Install the :ref:`pyocd-debug-host-tools` and make sure they are in your search
-        path.
+	Install the :ref:`linkserver-debug-host-tools` and make sure they are in your
+	search path.  LinkServer works with the default CMSIS-DAP firmware included in
+	the on-board debugger.
+
+        Linkserver is the default for this board, ``west flash`` and ``west debug`` will
```

Comment:
When I test this PR locally, Linkserver does not appear to be the defaut:

```
(.venv) ~/zephyr/zephyrproject/zephyr:(177314f17a...) ❯ west flash
-- west flash: rebuilding
[1/1] cd /home/danieldegrasse/zephyr/zephyrproject...ephyr/zephyrproject/zephyr/build/zephyr/zephyr.elf
-- west flash: using runner pyocd
WARNING: runners.pyocd: hex file (None) does not exist; falling back on .bin (/home/danieldegrasse/zephyr/zephyrproject/zephyr/build/zephyr/zephyr.bin). Consider enabling CONFIG_BUILD_OUTPUT_HEX.
-- runners.pyocd: Flashing file: /home/danieldegrasse/zephyr/zephyrproject/zephyr/build/zephyr/zephyr.bin
0001551 I Loading /home/danieldegrasse/zephyr/zephyrproject/zephyr/build/zephyr/zephyr.bin [load_cmd]
[==================================================] 100%
0002650 I Erased 0 bytes (0 sectors), programmed 0 bytes (0 pages), skipped 20480 bytes (5 pages) at 18.21 kB/s [loader]
(.venv) ~/zephyr/zephyrproject/zephyr:(177314f17a...) 
```
versus: 
```
west flash -r linkserver
-- west flash: rebuilding
[1/1] cd /home/danieldegrasse/zephyr/zephyrproject...ephyr/zephyrproject/zephyr/build/zephyr/zephyr.elf
-- west flash: using runner linkserver
-- runners.linkserver: LinkServer: /usr/local/LinkServer/LinkServer, version v1.3.15
INFO: Exact match for MK64FN1M0xxx12:FRDM-K64F found
INFO: Selected device MK64FN1M0xxx12:FRDM-K64F
INFO: Getting available probes
INFO: Selecting probe by index
INFO: Selected probe #1 02400b0128884e45004400106bf000478021000097969900 (DAPLink CMSIS-DAP
```